applications/portfolioDrafts/deletePortfolioDraft.ts

```diff
@@ -0,0 +1,46 @@
+import { DeleteCommand } from "@aws-sdk/lib-dynamodb";
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { parse } from "uuid";
+import { dynamodbClient as client } from "./utils/dynamodb";
+
+const TABLE_NAME = process.env.ATAT_TABLE_NAME;
+// const PRIMARY_KEY = process.env.PRIMARY_KEY || "";
+
+/**
+ * Deletes a Portfolio Draft
+ *
+ * @param event - The DELETE request from API Gateway
+ */
+export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+  const portfolioDraftId = event.pathParameters?.portfolioDraftId;
+  if (portfolioDraftId == null) {
```

Comment:
```suggestion
  if (!portfolioDraftId) {
```

To catch `undefined` or `""` as well?